sycl/include/sycl/ext/intel/experimental/esimd/simd_view.hpp

```diff
@@ -90,6 +90,9 @@ class simd_view : public detail::simd_view_impl<BaseTy, RegionTy> {
   DEF_RELOP(!=)
 
 #undef DEF_RELOP
+
+  // negation operator
+  auto operator!() { return *this == 0; }
```

Comment:
This code duplication will be cleaned up after https://github.com/intel/llvm/pull/4351

Reply:
OK. We will also need to add logical operator &&, || support, maybe with simd_mask patch.